src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlException.cs

```diff
@@ -82,67 +85,69 @@ public SqlErrorCollection Errors
             }
         }
 
-        /// <include file='..\..\..\..\..\..\..\doc\snippets\Microsoft.Data.SqlClient\SqlException.xml' path='docs/members[@name="SqlException"]/ClientConnectionId/*' />
+        /// <include file='../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlException.xml' path='docs/members[@name="SqlException"]/ClientConnectionId/*' />
         public Guid ClientConnectionId
         {
             get
             {
-                return this._clientConnectionId;
+                return _clientConnectionId;
             }
         }
 
+#if NETFRAMEWORK
         /*virtual protected*/
         private bool ShouldSerializeErrors()
         { // MDAC 65548
             return ((null != _errors) && (0 < _errors.Count));
         }
+#endif
 
-        /// <include file='..\..\..\..\..\..\..\doc\snippets\Microsoft.Data.SqlClient\SqlException.xml' path='docs/members[@name="SqlException"]/Class/*' />
+        /// <include file='../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlException.xml' path='docs/members[@name="SqlException"]/Class/*' />
         public byte Class
         {
             get { return Errors.Count > 0 ? Errors[0].Class : default; }
```

Comment:
Since you mentioned style, most of these getters can be simplified.

Reply:
sounds good, I'll do that in the next commit.